spec/routing/maintenance_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe "Maintenance Page" do
+  context "when the maintenance variable is set" do
+    before do
+      Rails.application.config.maintenance_mode = true
+      Rails.application.reload_routes!
+    end
+
+    after do
+      Rails.application.config.maintenance_mode = false
+      Rails.application.reload_routes!
+    end
+
+    it "routes to the maintenance controller" do
+      expect(get("/email/authenticate")).to route_to(controller: "maintenance", action: "show", "": "email/authenticate")
+    end
+
+    it "routes to the maintenance controller" do
+      expect(post("/email/authenticate")).to route_to(controller: "maintenance", action: "show", "": "email/authenticate")
+    end
+  end
+
+  context "when the maintenance variable is not set" do
+    it "routes to the usual controller" do
+      load Rails.root.join("config/initializers/maintenance.rb")
```

Comment:
This can just be `Rails.application.config.maintenance_mode = false` I think?

Reply:
Actually you might be able to get away without lines 24/25 at all. Since the other test puts things back as it found them (no maintenance message), this test should be okay without them.